bridgetegan.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: page
+title: b.e.
+---
+
+## What I do
+I work at [Puppet](www.puppet.com) as a partner enablement specialist. I read, hike, write, and bird. 
```

Comment:
Yay! you hike too! 
I love hiking. We flew over the Sierras yesterday. There's so much snow this year.